include/nci_types.h

```diff
@@ -129,6 +129,9 @@ typedef struct nci_mode_param_poll_a {
 /* Table 56: Specific Parameters for NFC-B Poll Mode */
 typedef struct nci_mode_param_poll_b {
     guint8 nfcid0[4];
+    guint8 app_data[4];     /* Since 1.1.3 */
+    guint8 prot_info[4];    /* Since 1.1.3 */
+    guint8 prot_info_len;   /* Since 1.1.3 */
```

Comment:
New fields have to be added to the end of the structure for ABI compatibility

Reply:
Like this:
```c
typedef struct nci_mode_param_poll_b {
￼     guint8 nfcid0[4];
     guint fsc;  /* FSCI converted to bytes */
     /* Since 1.1.3 */
￼     guint8 app_data[4];
￼     guint8 prot_info[4];
￼     guint8 prot_info_len;
￼ } NciModeParamPollB;
```

Reply:
And this would be even better (use `GUtilData` for buffers of variable size):
```c
typedef struct nci_mode_param_poll_b {
￼     guint8 nfcid0[4];
     guint fsc;  /* FSCI converted to bytes */
     /* Since 1.1.3 */
￼     guint8 app_data[4];
￼     GUtilData prot_info;
￼ } NciModeParamPollB;
```